pulpcore/pulpcore/app/viewsets/repository.py

```diff
@@ -115,12 +115,11 @@ class Meta:
 
 class ImporterViewSet(NamedModelViewSet):
     endpoint_name = 'importers'
-    lookup_field = 'name'
     serializer_class = ImporterSerializer
     queryset = Importer.objects.all()
     filter_class = ImporterFilter
 
-    def update(self, request, name, partial=False):
+    def update(self, request, pk, partial=False):
```

Comment:
It's sort of weird that the pk gets passed everywhere, but importer.pk is what is used. I don't think we should change anything, this isn't relevant to your work, but your work does highlight this weirdness.

Reply:
This code will change when my DRY up PR is merged. I am rebasing it right now.

Reply:
thanks @dkliban